web/src/composebox_typeahead.js

```diff
@@ -231,7 +231,7 @@ function handle_keydown(e) {
                 if (should_enter_send(e)) {
                     e.preventDefault();
                     if (
-                        compose_validate.warn_for_text_overflow_when_tries_to_send() &&
+                        !compose_validate.maybe_warn_for_text_overflow_on_send_attempt() &&
```

Comment:
I know sometimes our verifiers return false when they find an error, but that was really unclear to me in the `compose_typeahead` file, so I thought this change would be helpful.